infra/us-east-1/prod/rds/bastion/security-group/terragrunt.hcl

```diff
@@ -0,0 +1,40 @@
+locals {
+
+  # Automatically load environment-level variables
+  account_vars     = read_terragrunt_config(find_in_parent_folders("account.hcl"))
+
+  # Extract out common variables for reuse
+  project        = local.account_vars.locals.project
+  app            = "marlowe-runtime"
+
+  bastion_name          = "${local.project}-${local.app}-bastion"
+  bastion_key_pair_name = "${local.project}-${local.app}-db-bastion-kp"
+
+  tags = {
+    Owner        = "Terraform"
+    tribe        = "smart-contracts"
+    App          = "marlowe-runtime"
+  }
```

Comment:
Default tags are already set up on the provider (using default_tags parameter). Probably you don't need to declare these.